lib/datadog/tracing/contrib/active_support/notifications/event.rb

```diff
@@ -62,6 +64,21 @@ def payload_exception(payload)
                 payload[:exception_object] ||
                   payload[:exception] # Fallback for ActiveSupport < 5.0
               end
+
+              def on_start(_span, _event, _id, _payload); end
+
+              def on_finish(span, _event, _id, payload)
+                record_exception(span, payload)
+              end
```

Comment:
👍 These are making sure `method(:on_start)` and `method(:on_finish)` won't raise exception